plant/plants/api/serializers.py

```diff
@@ -44,18 +54,24 @@ class Meta:
             "owner",
             "staff",
             "staff_ids",
+            "family_id",
+            "family",
         ]
 
     def create(self, validated_data):
         staff = validated_data.pop("staff_ids")
+        family = validated_data.pop("family_id")
```

Comment:
Actually to save one line you can just do
validated_data["family"] = validated_data.pop("family_id")

and then L65 is unnecessary